wcbench.sh

```diff
@@ -480,11 +545,11 @@ uninstall_odl()
 {
     if [ -d $ODL_DIR ]; then
         echo "Removing $ODL_DIR"
-        rm -rf $ODL_DIR
+        sudo rm -rf $ODL_DIR
     fi
     if [ -f $ODL_ZIP_PATH ]; then
         echo "Removing $ODL_ZIP_PATH"
-        rm -f $ODL_ZIP_PATH
```

Comment:
I made my comment on the "removed" part of the diff, which I guess caused the confusion. I think it should be `rm -f $ODL_ZIP_PATH`, the PR makes it `sudo rm -rf $ODL_ZIP_PATH`